cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -704,6 +705,13 @@ def _get_connection_info(self, volume, backing, connector):
                 connection_info['data']['import_data'] = \
                     self._get_connection_import_data(volume)
 
+            # If we are migrating to this volume, we need to
+            # create a writeable handle for the migration to work.
+            if (volume['status'] == 'available' and
```

Comment:
Why not merging this with the previous `if`?

Reply:
the else in this case really doesn't matter, as the volume['status'] makes it exclusive anyway.

Reply:
I mean unfying the conditions in a single if to avoid the repetition of setting import_data.
But then it might be less readable.

Reply:
ok It will be less readable, but I'll combine them.